kubernetes/linux/main.sh

```diff
@@ -464,6 +470,15 @@ if [[ ( ( ! -e "/etc/config/kube.conf" ) && ( "${CONTAINER_TYPE}" == "Prometheus
       fi
 fi
 
+# If the prometheus sidecar isn't doing anything then there's no need to run mdsd and telegraf in it. 
+if [[ ( "${CONTAINER_TYPE}" == "PrometheusSidecar" ) && 
+      ( "${CUSTOM_PROM_MONITOR_PODS}" == "false" ) && 
+      ( "${OSM_CONFIGURATION_NAMESPACES_COUNT}" -eq 0 ) ]]; then
+      setGlobalEnvVar MUTE_PROM_SIDECAR true
```

Comment:
@jatakiajanvi12 , I think we should add MUTE_PROM_SIDECAR to the telemetry so that we can track how many clusters where the sidecar muted and also helpful for the CRI investigation? This should be straightforward just you need to write to bashrc similar to other env variables in main.sh.

Reply:
Added the telemetry in main.sh as discussed